app/assets/stylesheets/components/beta_tag/_beta_tag.scss

```diff
@@ -0,0 +1,11 @@
+.beta-tag {
+  @include inline-block();
+  @include body(18, 24);
+  font-weight: 700;
+  position: relative;
+  bottom: -$baseline-unit*5;
```

Comment:
When the beta_tag html is removed the margin on the `h1` returns to normal giving the spacing in the design without needing to change the styles for the `h1` or any elements above it. Or that's the idea.